### PR TITLE
Use 30-seconds-starter template

### PR DESCRIPTION
**Pull request by Chalarangelo**
Migration procedure to the new [common template for 30 seconds repositories](https://github.com/30-seconds/30-seconds-starter).

**This is still a work in progress, do not merge unless I explicitly say so!**

### Tasks

- [x] Update snippet structure to new format
- [x] Evaluate every snippet, check code, description, explanation, update as necessary
- [x] Cleanup existing repository, remove anything not necessary (including test system - at this time, we do not have enough resources to maintain this, but we can ensure better community curation going forward, so it is not necessary)
- [x] Copy files from the starter, configure accordingly
- [x] Update static parts, guidelines, issue/PR templates
- [x] Update Travis, add Netlify integration
- [x] Configure Google Analytics
- [x] Evaluate that all scripts (`extractor`, `builder`, `webber`) work as planned
- [x] Setup Netlify properly and merge, disable GitHub Pages (expect ~2 hours website downtime)

### CODE REVIEW

File: snippet_template.md [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
 def function_name(args):
-    # code
-    return 0
+  # code
+  return 0
 ```
 
 ``` python
-function_name(val) # result
+functionName(val) # result
```
Comment (fejes713) [inline snippet_template.md]: why not `function_name`? AFAIK Python uses that naming convention and not a camelCase

Comment (Chalarangelo) [reply snippet_template.md]: Fair point, I'll patch that up later.

**Comment by Chalarangelo** [comment]
**Update**: Existing snippets updated, I will work on a few new snippets.

**Comment by fejes713** [comment]
>  Evaluate that all scripts (extractor, builder, webber) work as planned

I think I am done with this. We just need to update favicon and syntax highlighting and we're done. 

So far I solved README generator with `extractor` and `builder`, copied and configured Gatsby files.

Gatsby config needs updated Google Analytics code.

**Comment by Chalarangelo** [comment]
I think we are missing a few lines in the `.gitignore`, diff is extraordinarily large, but nothing to worry about. Meanwhile, Google Analytics code should be the same as the 30-seconds-of-code website, as GA does not use different tracking IDs for subdomains, but separates their traffic in the analytics later, which seems to suit our purpose a whole lot better than I anticipated.

**Comment by Chalarangelo** [comment]
I'll check everything tomorrow before noon and add as many snippets as I can. We can probably migrate either tomorrow or on Thursday, downtime should be minimal.

**Comment by Chalarangelo** [comment]
## Chores complete, migrating as we speak, shouldn't be too long, hang on tight!

icon

**Comment by Chalarangelo** [comment]
**Migration complete, welcome to the new 30-seconds-of-python with all the bells and whistles of the unified repository structure. A huge thanks to @fejes713 for his help!**

**Pull request merged**